oclint-scripts/make

```diff
@@ -1,4 +1,4 @@
 #! /bin/sh -e
 
-./ci -reset
-./ci -setup -release
+./ci -reset "$@"
+./ci -setup -release "$@"
```

Comment:
This script is mainly for people who want to build oclint local releases. I still want to understand the reason to override this setting.